maquetado/scss/titles.scss

```diff
@@ -0,0 +1,29 @@
+.tech-area-title {
+  align-items: center;
+  display: flex;
+  margin: 0 0 30px;
+  position: relative;
```

Comment:
If the pseudo-element has position `relative`, there's no need to add this relative 😄